system/keymaps/remote.xml

```diff
@@ -577,16 +577,28 @@
     <remote>
       <left>StepBack</left>
       <right>StepForward</right>
-      <up>ChannelUp</up>
-      <down>ChannelDown</down>
+      <up>Up</up>
+      <down>Down</down>
+      <skipplus>SkipNext</skipplus>
+      <skipminus>SkipPrevious</skipminus>
+      <pageup>ChannelUp</pageup>
+      <pagedown>ChannelDown</pagedown>
+      <channelplus>ChannelUp</channelplus>
+      <channelminus>ChannelDown</channelminus>
```

Comment:
If we remove these two lines the feature simply will not work unless the user inserts these two lines into his config. The most natural thing on earth, imo, is to map channelup key to channel up (down resp.) as default.